test/AliceGun_cfg.py

```diff
@@ -0,0 +1,189 @@
+import os
```

Comment:
check whether needed 
if yes clean up and use generic name
think about unittest integration for heppy / papas cpp. getting a perfect match could be a prerequisite for a PR to one of them?